resources/java_memory_assistant/bin/clean-up.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
```

Comment:
I'm not sure that I buy that agents are better if they have less code (APM agents are a compelling counter argument).  You've not actually reduced the amount of complexity of your agent, instead you've moved that complexity from a language with very robust testing facilities (Java) to a language with lesser test facilities (shell scripts).

But that being said, since this is the contribution you're comfortable with we'll stick with it.